src/subcommand/init/shell/powershell.rs

```diff
@@ -0,0 +1,84 @@
+use super::{HookConfig, ShellConfig};
+
+use anyhow::Result;
+
+use std::borrow::Cow;
+
+pub const CONFIG: ShellConfig = ShellConfig {
+    z,
+    alias,
+    hook: HookConfig {
+        prompt: HOOK_PROMPT,
+        pwd: hook_pwd,
+    },
+};
+
+fn z(z_cmd: &str) -> String {
```

Comment:
`--z-cmd` has been renamed to simply `--cmd`. Could you rename the instances of `z_cmd` here?

Reply:
Fixed (code was copied from `fish.rs` so I updated that file as well.